regen-network/regen-web
# chore: update usage of project handle to slug

## Description

Closes: #2113
Needs: https://github.com/regen-network/regen-server/pull/382



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

To make sure slug-based project queries are still working ok, visit https://deploy-preview-2129--regen-marketplace.netlify.app/project/wilmot

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
